lhotse/kaldi.py

```diff
@@ -1,12 +1,32 @@
 import warnings
 from collections import defaultdict
 from pathlib import Path
-from typing import Any, Dict, Optional, Tuple
+from typing import Any, Dict, Optional, Tuple, NamedTuple
 
 from lhotse import CutSet, FeatureSet, Features, Seconds
 from lhotse.audio import AudioSource, Recording, RecordingSet
 from lhotse.supervision import SupervisionSegment, SupervisionSet
 from lhotse.utils import Pathlike, is_module_available
+from lhotse.audio import audioread_info 
+
+
+def from_file(
```

Comment:
This function is to get the duration of a sound file.
`get_duration` maybe is a better name than `from_file`.

Reply:
I agree with you. Done!